src/BlazorUI/Bit.BlazorUI.Tests/Buttons/BitButtonTests.cs

```diff
@@ -156,7 +157,7 @@ public void BitButtonAriaHiddenTest(bool ariaHidden, bool expectedResult)
 
         var bitButton = com.Find(".bit-btn");
 
-        Assert.AreEqual(bitButton.HasAttribute("aria-hidden"), expectedResult);
+        Assert.AreEqual(expectedResult, bitButton.HasAttribute("aria-hidden"));
```

Comment:
Normally expected comes first then actual.

I can also go through the tests and align this everywhere.

Reply:
tnx, that would be great 🚀

Reply:
Said and done.